use_case_guide/real_time_dashboards.rst

```diff
@@ -111,15 +117,15 @@ for each of the last 30 days.
 .. code-block:: sql
 
   CREATE TABLE http_request_1min (
-        site_id INT,
-        ingest_time TIMESTAMPTZ, -- which minute this row represents
-
-        error_count INT,
-        success_count INT,
-        request_count INT,
-        average_response_time_msec INT,
-        CHECK (request_count = error_count + success_count),
-        CHECK (ingest_time = date_trunc('minute', ingest_time))
+    site_id INT,
+    ingest_time TIMESTAMPTZ, -- which minute this row represents
+
+    error_count INT,
+    success_count INT,
+    request_count INT,
+    average_response_time_msec INT,
+    CHECK (request_count = error_count + success_count),
+    CHECK (ingest_time = date_trunc('minute', ingest_time))
```

Comment:
By the way, hats off to whoever thought of these CHECK constraints. Great use for them. 😄